website/content/en/docs/setup/installation/operating-systems/nixos.md

```diff
@@ -1,14 +1,24 @@
 ---
 title: Install Vector on NixOS
 short: NixOS
-supported_installers: ["Nix", "Vector installer", "Docker"]
 weight: 6
```

Comment:
We could leave this but drop `Vector installer` since that one won't work on Nix.